packages/next/src/build/index.ts

```diff
@@ -1961,8 +1962,19 @@ export default async function build(
                     'next/dist/server/lib/render-server-standalone'
                   )
                 : null,
+              require.resolve(
+                'next/dist/compiled/minimal-next-server/next-server-cached.js'
+              ),
+              require.resolve(
+                'next/dist/compiled/minimal-next-server/next-server.js'
+              ),
```

Comment:
Should we have this behind an experimental config so that we only trace the relevant one as we shouldn't need to continue tracing the full `next-server` if we're using the `minimal-next-server`.